packages/react-dom/src/__tests__/ReactServerRendering-test.js

```diff
@@ -1153,5 +1153,11 @@ describe('ReactDOMServer', () => {
       );
       expect(output).toBe(`<my-custom-element></my-custom-element>`);
     });
+    it('Style property with empty values should be omitted.', () => {
```

Comment:
Should they? Is this always correct?

Reply:
Gotcha. Good call copying the comment so that we're at least consistent across server and client.